ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Homebrew_intel/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Install Intel Homebrew onto our arm64 mac machines. This is needed to build x64 jdk8
+# The freetype build process requires Intel libpng installed, which needs Intel Homebrew
+# See https://github.com/adoptium/infrastructure/issues/2536#issuecomment-1708716478
+
+- name: Check if Intel Homebrew is installed
+  stat:
+    path: /usr/local/Homebrew/bin/brew
+  register: intel_homebrew_installed
+
+- name: Run Homebrew install script in a Rosetta shell
+  ansible.builtin.shell: yes | arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+  when: not intel_homebrew_installed.stat.exists
+
+- name: Install Intel libpng
+  ansible.builtin.shell: arch -x86_64 yes | /usr/local/Homebrew/bin/brew install libpng
```

Comment:
This is presumably the bit that caused the problems from https://github.com/adoptium/infrastructure/issues/3231 so I'm going to block this change on the basis that we presumably fix this PR before it goes live.